base tailwind preset / config for expo and next?

Hey there, i think the template should use base preset / config because we developing the same thing in web and android / ios. just a suggestion : )

hi @juliusmarminge, i have minor problem here can you help me?. When iam requiring postcss config into nextjs it throws this error:
```
error - ../../node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[3].oneOf[8].use[1]!../../node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[3].oneOf[8].use[2]!./src/styles/globals.css
Error: Cannot find module '@acme/postcss.js'
```
it seems package `@acme/tailwind-config` not include postcss config

--- a/apps/expo/package.json
+++ b/apps/expo/package.json
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@acme/api": "*",
+    "@acme/tailwind-config": "*",
     "@shopify/flash-list": "^1.3.1",
     "@tanstack/react-query": "^4.14.1",
     "@trpc/client": "10.0.0-rc.4",

--- a/apps/expo/tailwind.config.cjs
+++ b/apps/expo/tailwind.config.cjs
@@ -1,8 +1,1 @@
-/** @type {import('tailwindcss').Config} */
-module.exports = {
-  content: ["./src/**/*.{js,ts,jsx,tsx}", "./src/_app.tsx"],
-  theme: {
-    extend: {},
-  },
-  plugins: [],
-};
+module.exports = require("@acme/tailwind-config");

--- a/apps/nextjs/package.json
+++ b/apps/nextjs/package.json
@@ -13,6 +13,7 @@
     "@acme/api": "*",
     "@acme/auth": "*",
     "@acme/db": "*",
+    "@acme/tailwind-config": "*",
     "@tanstack/react-query": "^4.14.1",
     "@trpc/client": "10.0.0-rc.4",
     "@trpc/next": "10.0.0-rc.4",

--- a/apps/nextjs/postcss.config.cjs
+++ b/apps/nextjs/postcss.config.cjs
@@ -1,6 +1,1 @@
-module.exports = {
-  plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-  },
-};
+module.exports = require("@acme/tailwind-config/postcss.config.js")

--- a/apps/nextjs/tailwind.config.cjs
+++ b/apps/nextjs/tailwind.config.cjs
@@ -1,8 +1,1 @@
-/** @type {import('tailwindcss').Config} */
-module.exports = {
-  content: ["./src/**/*.{js,ts,jsx,tsx}"],
-  theme: {
-    extend: {},
-  },
-  plugins: [],
-};
+module.exports = require("@acme/tailwind-config");

--- a/packages/config/tailwind-config/index.js
+++ b/packages/config/tailwind-config/index.js
@@ -1,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{js,ts,jsx,tsx}", "./src/_app.tsx"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};

--- a/packages/config/tailwind-config/package.json
+++ b/packages/config/tailwind-config/package.json
@@ -1,0 +1,11 @@
+{
+  "name": "@acme/tailwind-config",
+  "version": "0.1.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "autoprefixer": "^10.4.7",
+    "postcss": "^8.4.14",
+    "tailwindcss": "^3.1.8"
+  }
+}

--- a/packages/config/tailwind-config/postcss.config.js
+++ b/packages/config/tailwind-config/postcss.config.js
@@ -1,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+  },
+};

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -4,3 +4,4 @@ packages:
   - packages/api
   - packages/auth
   - packages/db
+  - packages/config/tailwind-config

